Functions.Templates/Resources/Resources.resx

```diff
@@ -1747,4 +1675,40 @@ function app.</value>
   <data name="temp_category_durableFunctions" xml:space="preserve">
     <value>Durable Functions</value>
   </data>
+  <data name="BlobTrigger_name" xml:space="preserve">

```

Comment:
Can't use resource strings for names, it breaks CLI, you see $values in the template names when you type func new, I am following up on that and am planning to remove the localization in 4 application insights templates that use resource strings in names

Reply:
Many function templates in dev have resource strings for names. How will they be localized without the resx?